hangupsbot/plugins/default.py

```diff
@@ -48,7 +48,18 @@ def echo(bot, event, *args):
 
         yield from command.run(bot, event, *["convecho", "id:" + convid, _text])
 
-
+def admins(bot,event,*args):
+    admin_list=bot.get_config_option('admins')
+    text="<b><u>List of Admins</u></b><br />"
+    for admin_id in admin_list:
+        user_object = bot.get_hangups_user(admin_id)
+        if user_object.is_self:
+            continue
+        fullname=user_object.full_name
+        text+="{} <br />".format(fullname)
+    yield from bot.coro_send_message(event.conv,text)
```

Comment:
avoid `+=` string concatenation and use pythonic `"<separator>".join(<list>)` instead